src/components/overlays/SavePrompt.tsx

```diff
@@ -0,0 +1,48 @@
+import classnames from 'classnames';
+import _ from 'lodash';
+import React from 'react';
+import { FiArrowRight } from 'react-icons/fi';
+import { getLabel } from '~/codeLists/labelCodes';
+import * as s from './savePrompt.scss'; // should use savePrompt.scss
+
+const SavePrompt = (props: any) => {
+    const changesContent = () => {
+        const current = _.cloneDeep(props.current);
+        const old = _.cloneDeep(props.old);
+        const divs = [];
+
+        for (const property in current) {
+            const linkAttribute = current[property];
+            const oldLinkAttribute = old[property];
+            if (_.isEqual(linkAttribute, oldLinkAttribute)) {
+                delete old[property];
+                delete old[property];
+            }
+        }
+
+        for (const property in old) {
+            const propertyLabel = getLabel(props.type, property);
+            divs.push(
+                <div className={classnames(s.formItem, s.savePromptRow)}>
```

Comment:
Editing link geometry causes a bug " Each child in an array or iterator should have a unique "key" prop". This prob is because of you are rendering rows in a loop without specifying a key for a row. One way to fix this is to use: <div key={`row-${index}`}></div> (index comes from map: ```return properties.map((property: string, index: number) => {```